rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json

```diff
@@ -0,0 +1,3105 @@
+                                                    ],
+                                                    "minimum_outdoor_airflow": 1769.8029119999994,
+                                                    "maximum_outdoor_airflow": 1769.8029119999994,
+                                                    "air_energy_recovery": {
+                                                        "id": "Air Energy Recovery 1",
+                                                        "energy_recovery_operation": "WHEN_MINIMUM_OUTSIDE_AIR"
+                                                    }
+                                                }
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "boilers": [
+                            {
+                                "id": "Boiler 1",
+                                "loop": "Boiler Loop 1",
+                                "energy_source_type": "NATURAL_GAS"
+                            }
+                        ],
+                        "chillers": [
+                            {
+                                "id": "Chiller 1",
+                                "cooling_loop": "Chiller Loop 1"
+                            }
+                        ],
+                        "pumps": [
+                            {
+                                "id": "Boiler Pump 1",
+                                "loop_or_piping": "Boiler Loop 1",
+                                "speed_control": "FIXED_SPEED"
+                            },
+                            {
+                                "id": "Chiller Pump 1",
+                                "loop_or_piping": "Chiller Loop 1",
+                                "speed_control": "FIXED_SPEED"
+                            },
+                            {
+                                "id": "Secondary CHW Pump",
+                                "loop_or_piping": "Secondary CHW Loop 1",
+                                "speed_control": "VARIABLE_SPEED"
+                            }
+                        ],
+                        "fluid_loops": [
+                            {
+                                "id": "Boiler Loop 1",
+                                "type": "HEATING"
+                            },
+                            {
+                                "id": "Chiller Loop 1",
+                                "type": "COOLING",
+                                "child_loops": [
+                                    {
+                                        "id": "Secondary CHW Loop 1",
+                                        "type": "COOLING"
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ],
```

Comment:
The ```schedules``` key's missing. Could you add proper schedules here and also in other cases?

Reply:
I believe we still need to have the ```schedules``` to find the max heating thermostat setpoint values. What's not clear to me the schema doesn't specify what constant value it has to be. I believe the description should be updated.

Reply:
I interpreted it to mean if `thermostat_heating_setpoint_schedule` or `thermostat_cooling_setpoint_schedule` are not provided, then `design_thermostat_heating_setpoint` or `design_thermostat_cooling_setpoint` are assumed to be constant

Reply:
Yeah good point! The thing is RDS doesn't specify what the constant value should be when the schedule doesn't exist. Let me ask this question to the RDS team (I'll copy you).